build.sh

```diff
@@ -24,12 +24,8 @@ for d in $PLUGINS; do
 	if [ -d "$d" ]; then
 		plugin="$(basename "$d")"
 		echo "  $plugin"
-		# use go install so we don't duplicate work
-		if [ -n "$FASTBUILD" ]
-		then
-			GOBIN=${PWD}/bin go install -pkgdir $GOPATH/pkg "$@" $REPO_PATH/$d
-		else
-			go build -o "${PWD}/bin/$plugin" -pkgdir "$GOPATH/pkg" "$@" "$REPO_PATH/$d"
-		fi
+		go build -o "${PWD}/bin/$plugin" -pkgdir "$GOPATH/pkg" "$@" "$REPO_PATH/$d" &
 	fi
 done
+
+wait
```

Comment:
The function of `wait` is described as follows in `man sh`:
> wait for all jobs to complete and the return an exit status of zero.

We would need to know if any of the background processes failed.

Reply:
They should still be reported and not cause any stalls. For example:
```
echo "test" &
sleep 2
ls /t &
sleep 5 &
echo "test"
wait 
```
`ls /t &` will fail for me but everything is still waited upon (and executed) as expected.

Reply:
we want to get the failed status code reported up to the caller of `build.sh`